R/install.R

```diff
@@ -276,7 +276,10 @@
 #'     be old.
 #' @param version `character(1)` _Bioconductor_ version to install,
 #'     e.g., `version = "3.8"`. The special symbol `version = "devel"`
-#'     installs the current 'development' version.
+#'     installs the current 'development' version. If a user has
+#'     installed a new version of R that is incompatible with their
+#'     Bioconductor installation, setting this to a compatible version
```

Comment:
@Alanocallaghan This isn't always the case since the user can have R-devel
installed, for example, and that version not compatible with either version of
Bioconductor..
It's a tricky situation that we further explain the vignette and website page..

Reply:
Well I hedged slightly saying "a compatible version" rather than "a newer version" for that reason. Just wanted to illustrate the point.

Reply:
We'd be satisfied with referring the user to the vignette for troubleshooting.. it should say something like: "In the case of incompatible R and Bioconductor versions, see advanced usage in the vignette."